projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -1052,29 +1122,28 @@ protected function _get_coalesced_range( array $date_range ) {
 	 * Generate and output ajax response for related posts API call.
 	 * NOTE: Calls exit() to end all further processing after payload has been outputed.
 	 *
-	 * @param array $excludes array of post_ids to exclude
+	 * @param array $excludes array of post_ids to exclude.
 	 * @uses send_nosniff_header, self::get_for_post_id, get_the_ID
-	 * @return null
 	 */
-	protected function _action_frontend_init_ajax( array $excludes ) {
+	protected function action_frontend_init_ajax( array $excludes ) {
 		define( 'DOING_AJAX', true );
 
-		header( 'Content-type: application/json; charset=utf-8' ); // JSON can only be UTF-8
+		header( 'Content-type: application/json; charset=utf-8' ); // JSON can only be UTF-8.
 		send_nosniff_header();
 
 		$options = $this->get_options();
 
-		if ( isset( $_GET['jetpackrpcustomize'] ) ) {
+		if ( isset( $_GET['jetpackrpcustomize'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Comment as to why this is safe?

Reply:
Left comments addressing the comments about comments 🪀 🐶 

I think as far as I can see they're safe to ignore, but happy to discuss further if you think otherwise.

Reply:
At a quick glance they do look like they're safe to ignore. But personally I'd change the comments you added to clearly say _why_ (p9dueE-4z8-p2) it's safe (i.e. as far as I can tell, this whole class is all view logic that doesn't change any options or do any DB writes) rather than just reiterating what the code is doing.